cmd/snap-update-ns/utils.go

```diff
@@ -108,6 +115,64 @@ func secureMkdirAll(name string, perm os.FileMode, uid, gid int) error {
 	return nil
 }
 
+func designWritableMimic(dir string) ([]*Change, error) {
```

Comment:
this function name sounds a bit strange to me...  alternatives could be: planWritableMimic, writableMimicPlan, writableMimicChanges

Reply:
+1